Sources/TemplateKit/Tag/DateFormat.swift

```diff
@@ -0,0 +1,15 @@
+import Async
+import Foundation
+
+public final class DateFormat: TagRenderer {
```

Comment:
This is not _a_ date format, this is a specific date format. I think the tag renderer should be named as such

Reply:
you can pass in any date format as the second parameter